src/editor/components/scenegraph/AIChatPanel.js

```diff
@@ -1218,6 +1238,52 @@ const AIChatPanel = () => {
                 entity.removeChild(segmentEl);
 
                 console.log('Removed segment at segmentIndex', segmentIndex);
+              } else if (call.name === 'projectInfoUpdate') {
+                // Handle project info update
+                const args = call.args;
+
+                // Get the project info entity
+                let projectInfoEntity = document.querySelector(
+                  '#project-info-entity'
+                );
+
+                // If the entity doesn't exist, create it
+                if (!projectInfoEntity) {
+                  console.log('Creating project info entity');
+                  projectInfoEntity = document.createElement('a-entity');
+                  projectInfoEntity.setAttribute('id', 'project-info-entity');
+                  projectInfoEntity.setAttribute('project-info', {});
+
+                  // Add to the scene under user layers
+                  const userLayers =
+                    document.querySelector('#user-layers') ||
+                    document.querySelector('a-scene');
```

Comment:
Why do you have a fallback to a-scene here? #user-layers should always exists right?

Reply:
yeah it's actually a mistake should be '#street-container' only, thanks for catching